test/common/http/dispatcher_test.cc

```diff
@@ -1273,100 +1273,6 @@ TEST_F(DispatcherTest, StreamResetAfterOnComplete) {
   ASSERT_EQ(cc.on_cancel_calls, 0);
 }
 
-TEST_F(DispatcherTest, ResetStreamLocalHeadersRemoteRaceLocalWins) {
```

Comment:
What was the failure here? Shouldn't the race between local and remote reset still be covered?

Reply:
There is no more race - because of serialization, it's just a matter of ordering, which is covered elsewhere. The specific failure here was that the synchronization point no longer gated what it originally did - cleanup happened before it instead of after. But it's also no longer relevant to the implementation. And the test would no longer be meaningful for what it was covering if I were to simply move the sync point.